src/ethereum_test_tools/common/types.py

```diff
@@ -1005,3 +1070,19 @@ def default(self, obj):
             return f
         else:
             return super().default(obj)
+
+
+def even_padding(input: Dict, excluded: List[Any | None]) -> Dict:
+    """
+    Adds even padding to each field in the input (nested) dictionary.
+    """
+    for key, value in input.items():
+        if key not in excluded:
+            if isinstance(value, dict):
+                even_padding(value, excluded)
+            elif value != "0x":
+                strip = value.lstrip("0x")
+                new_value = "0" if strip == "" else strip
+                if len(new_value) % 2:
+                    input[key] = "0x0" + new_value
```

Comment:
I think this statement should have an else (adding only "0x"), unless I misunderstood something.

Reply:
Sounds good - I'll add the else for sanity!